src/main/java/core/basesyntax/report/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.report.report;
+
+import core.basesyntax.model.FruitOperation;
+import java.util.List;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String DEFAULT_DELIMITER = ",";
+    private static final String HEADER = "fruit" + DEFAULT_DELIMITER + "quantity";
+    private final StringBuilder stringBuilder;
+
+    public ReportGeneratorImpl() {
+        stringBuilder = new StringBuilder();
+        stringBuilder.append(HEADER);
+    }
+
+    @Override
+    public String getReport(List<FruitOperation> store) {
+        for (FruitOperation fruitOperation : store) {
+            stringBuilder.append(System.lineSeparator())
+                    .append(fruitOperation.getOperation())
+                    .append(DEFAULT_DELIMITER)
+                    .append(fruitOperation.getQuantity());
+        }
+        return stringBuilder.toString();
+    }
+}
```

Comment:
The `ReportGeneratorImpl` class should be placed in a package that reflects its purpose. Consider moving it to a package like `service.impl` as it is an implementation of a service.